salt/utils/files.py

```diff
@@ -616,6 +616,31 @@ def safe_filepath(file_path_name, dir_sep=None):
     return path
 
 
+def is_encoding(path, encoding="utf-8"):
+    '''
+    Detect if the file can be successfully decoded by the passed encoding
+
+    Args:
+
+        fp_ (pointer): A pointer to the file to check
```

Comment:
Shouldn't the arg name be `path` here? Also, there is no such type in Python as `pointer`, this should be `str` instead.